### PR TITLE
Fix IPV6 visualizations

### PR DESCRIPTION
**Pull request by Tostti**
### Description
This PR aims to fix the visualization of IPV6 addresses in different places of the application.

Changes:
- Add the `tableLayout='auto'` to some tables. This causes the column width to be auto-sized dynamically.

Additional changes:
- Add mocked API responses to test the related changes.
 
### Issues Resolved
https://github.com/wazuh/wazuh-kibana-app/issues/5464

### Evidence
- Agents section
image

- Agents section (reduced window width)
image

- Agents/{agent_id}/Inventory data
image

- Management/Cluster/Nodes
image

- Agents/{agent_id}
image

- Management/Groups/{group_name}/Agents
image

- Management/Configuration/Global Configuration/Remote
image

### Test
[wka-pr-5471-tests.md](https://github.com/wazuh/wazuh-kibana-app/files/11573525/wka-pr-5471-tests.md)

> This pull request adds some mocked API responses useful to the tests.

### Additional information
Part of this issue was already fixed in https://github.com/wazuh/wazuh-kibana-app/issues/4566 to be released in 4.5, I used that fix and added new functionalities

### CODE REVIEW

File: public/components/common/tables/table-with-search-bar.tsx [hunk 1]
```diff
@@ -79,31 +81,46 @@ export function TableWithSearchBar({
...
+            filters,
+            pagination,
+            sorting,
+          );
+          if (compressipv6) {
+            items.forEach(item => {
+              if (item.ip) {
+                item.ip = compressIPv6(item.ip);
+              }
+            });
```
Comment (Desvelao) [inline public/components/common/tables/table-with-search-bar.tsx:100]: **nitpick**: This seems to be too specific to add a property to replace the `ip` property of the response to be included in the reusable component. The component has a way to interact with the response item data, this is the `mapResponseItem` component property.

**suggestion**: You have 2 options:
1. if you want to replace the value for a property of the response, you can use the `mapResponseItem` property method of the TableWzAPI component, which transforms the response item.
```tsx
<TableWzAPI
  ...
  mapResponseItem={(item) => ({
    ...item,
    ip: compressIPv6(item.ip)
  })
}
/>
```

2. if you want to keep the original value (for some reason), you can use the `render` of the table column to render what you want.

I think option 1 is the best one for this case.

**Comment by yenienserrano** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| The columns of Agents table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The columns of Management/Groups/{group_name}/Agents table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The columns of Agents/{agent_id}/Inventory data table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The columns of Management/Cluster/Nodes table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The columns of Management/Global configuration/Remote table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The columns of Management/Configuration/Client table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |
| The IP address value is rendered correctly in Agents welcome. Use IPv4 and IPv6 addresses. | :black_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:black_circle: The columns of Agents table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: The columns of Management/Groups/{group_name}/Agents table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: The columns of Agents/{agent_id}/Inventory data table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: The columns of Management/Cluster/Nodes table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: The columns of Management/Global configuration/Remote table are dynamically sized according to their content. Use combinations of IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: The IP address value is rendered correctly in Agents welcome. Use IPv4 and IPv6 addresses.</summary>

Chrome - :black_circle:

image

Firefox - :black_circle:

Safari - :black_circle:

</details>

**Review by yenienserrano — APPROVED** [review]
CR: ✔️ 
Test: ✔️

**Comment by Desvelao** [comment]
Changes:

- Solved conflicts

**Review by yenienserrano — APPROVED** [review]

**Comment by Desvelao** [comment]
# Changes

- Moved changelog entry to 4.4.5

**Review by yenienserrano — APPROVED** [review]

**Review by Desvelao — APPROVED** [review]

**Pull request merged**